format.sh

```diff
@@ -51,13 +51,15 @@ YAPF_EXCLUDES=(
     '--exclude' 'sky/skylet/providers/gcp/**'
     '--exclude' 'sky/skylet/providers/azure/**'
     '--exclude' 'sky/skylet/providers/ibm/**'
+    '--exclude' 'sky/skylet/providers/oci/**'
```

Comment:
nit: since the OCI support is added for SkyPilot, it would be better to use yapf to format it, i.e., keep the formatting align with our main code. : )